experiments/mcp-eval/tasks/bigquery-1-dataset/task.yaml

```diff
@@ -0,0 +1,5 @@
+script:
+- prompt: "You need to create a KCC BigQueryDataset resource named 'my_bq_dataset' with a default table expiration of 3600 seconds and location 'US'. Use the available tools to generate the appropriate YAML manifest and then write it to the file at $(git rev-parse --show-toplevel)/.build/tasks/bigquery-1-dataset/dataset.yaml. Hint: Use the available MCP tools to find the resource schema and generate the YAML."
```

Comment:
I suggest that the eval should only be the first sentence.  Most of the rest can go into the system prompt / tool descriptions.  The output file path feels like something you can auto-add if that's how you choose to capture the eval output.

Reply:
Agree! I'll do that